features/pageobjects/alert.page.js

```diff
@@ -0,0 +1,41 @@
+const { $ } = require('@wdio/globals')
+const Page = require('./page');
+
+
+class AlertPage extends Page {
+
+    get JSAlert() {
+        return $('//button[text()="Click for JS Alert"]');

```

Comment:
You locate all buttons on this page using text inside of button. You can parametrize a get function to pass a button text as a parameter and then nest it into xpath